### DISCUSSION TITLE
単純なラテックスコマンドを適用する

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1834] [Q&A] [answered]
単純なLaTexコマンドを適用しようとしても機能しません。

.reファイル内でLaTexを使用する方法を教えてもらえますか？

![image](https://user-images.githubusercontent.com/49851974/178907424-d96ede91-18a5-48bd-879b-e3079fb524ff.png)
![image](https://user-images.githubusercontent.com/49851974/178907691-dc6f1c06-043f-42ad-b8ca-de63b59d810a.png)

**Comment by kdmsnr** [comment, marked as answer]
不要な`\`が入っているのだと思います。

```
//embed[latex]{
\begin{center}
{Sample Text Label to be center}
\includegraphics{images/cover.jpg}
\end{center}
//}
```

**Reply by jasondee1992** [reply]
kdmsnrさん、ありがとうございます
